src/test/java/com/wikia/webdriver/common/core/EmailUtils.java

```diff
@@ -20,13 +20,24 @@
 
 import com.wikia.webdriver.common.logging.PageObjectLogging;
 
-public class MailFunctions {
+public class EmailUtils {
 
-  private MailFunctions() {
+  private static final String EMAIL_LINK_PATTERN = ".*<a[^>]*href=\"(?<url>[^\"]+?)\"[^>]+>%s</a>.*";
+  private static final String PASSWORD_RESET_LINK = String.format(EMAIL_LINK_PATTERN, "SET NEW PASSWORD");
+  private static final String CONFIRM_EMAIL_LINK = String.format(EMAIL_LINK_PATTERN, "Confirm Now");
+
+  // Pattern.DOTALL flag forces dot in regex to also match line terminators
+  private static Pattern getPatternForString(String value) {
+    return Pattern.compile(value, Pattern.DOTALL);
+  }
+
+  private EmailUtils() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)